apps/mobile/src/libs/components/background/components/planet/planet.tsx

```diff
@@ -0,0 +1,51 @@
+import { type StyleProp, type TextStyle } from "react-native";
+import LinearGradient from "react-native-linear-gradient";
+
+import { GradientColor } from "~/libs/enums/app/gradient-color.enum";
+
+import { styles } from "./styles";
+
+type Properties = {
+	color: keyof typeof colorToGradientMap;
+	direction?: keyof typeof gradientDirectionMap;
+	size: keyof typeof sizeToWidthHeightMap;
+	style?: StyleProp<TextStyle>;
+};
+
+const Planet = ({
+	color,
+	direction = "leftToRight",
+	size,
+	style,
+}: Properties) => {
+	const { end, start } = gradientDirectionMap[direction];
+
+	return (
+		<LinearGradient
+			colors={colorToGradientMap[color]}
+			end={end}
+			start={start}
+			style={[styles.bubble, sizeToWidthHeightMap[size], style]}
+		/>
+	);
+};
+
+const sizeToWidthHeightMap = {
```

Comment:
Please move mappers to a separate file, as we made for Text (don't create 3 separate files, I think we can declare them and export in maps.ts)